### DISCUSSION TITLE
<title>How can I be allowed to not "verify device" via email?

### DISCUSSION THREAD

**Discussion by eimrine** [community/community#53971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have an annoyance because GitHub makes me to check my email every time I log in and type the six digits, this process is called as "verify device". It happens maybe 99 times from 100 log in attempts. I use to clear cookies all the time and most of devices I use for accessing GitHub web interface is borrowed. I am not able to use other log in options (I do not carry any phone with me for the sake of privacy) so replacing "verifying device" to 2FA seems like not an option.
I would like even to pay a little bit for that absence of annoyance, just stop this security theater measure to my account, please.

**Comment by airtower-luna** [comment]
> I am not able to use other log in options (I do not carry any phone with me for the sake of privacy) so replacing "verifying device" to 2FA seems like not an option.

TOTP (that's what an "authenticator app" does) or a security key don't need a network connection, so maybe carrying a small, no-radio device that can do either of those is an option? :thinking: 

> most of devices I use for accessing GitHub web interface is borrowed

That seems like all the more reason to use 2FA, really, only one of those borrowed devices needs to be compromised (or malicious to begin with) to get your password stolen.
